src/uzal_cost.jl

```diff
@@ -8,14 +8,15 @@ export uzal_cost
 """
     uzal_cost(Y; kwargs...) → L
 Compute the L-statistic `L` according to Uzal et al.[^Uzal2011],
-for a phase space trajectory `Y` (timeseries or `Dataset`). The L-statistic is
+for a phase space trajectory `Y` (`Dataset`). To convert an `Array` into Dataset
+format, simply use `Dataset(Array)`. The L-statistic is
```

Comment:
THere is no reason to explain here how to convert an `Array` in to a dataset. the documentation of `Dataset` takes care of that.